src/app/client/src/app/modules/explore-page/components/explore-page/explore-page.component.ts

```diff
@@ -130,8 +134,21 @@ export class ExplorePageComponent implements OnInit, OnDestroy, AfterViewInit {
                 this.userType = result;
             }
         });
+
+        this.themeSubscription = this.theme$.subscribe((theme) => {
```

Comment:
what is the use of this line of code I am not seeing and use 
   this.themeSubscription = this.theme$.subscribe((theme) => {
        });

Reply:
As far as my knowledge we should subscribe the behavioursubject just like other sevices